rest-api-spec/src/main/resources/rest-api-spec/test/range/10_basic.yml

```diff
@@ -368,3 +332,85 @@ setup:
         body: { "size" : 0, "query" : { "range" : { "date_range" : { "gte": "2017-09-03", "lte" : "2017-09-04", "relation": "within" } } } }
 
   - match: { hits.total: 0 }
+
+---
+"Query on range fields in query_string":
+
+  - skip:
+      version:  " - 6.1.0"
```

Comment:
I think this should be " - 6.0.99"? It looks like the feature is in >= 6.1.0, not > 6.1.0?